### PR TITLE
More python examples

### PR DESCRIPTION
**Pull request by gregorni**

### CODE REVIEW

**Comment by gregorni** [comment]
So after talking with @sonnyp on Matrix, I'm going to do one PR per demo. I'd like to keep this list here though, so that I can keep track of them myself.

**Comment by theCapypara** [comment]
In that case I will mark this as draft and remove reviews. Please close this when you don't need it anymore.

**Comment by gregorni** [comment]
> In that case I will mark this as draft and remove reviews. Please close this when you don't need it anymore.

Ah, you don't have to do that, just merge ahead. I'll find this either way.

**Comment by theCapypara** [comment]
Wait, but you said you were going to open separate PRs. In that case this PR is not meant to be merged.

**Comment by gregorni** [comment]
Well, we said since it's just these five demos, it's fine, but with everything else I'll do per-demo PRs.

File: src/Library/demos/Avatar/main.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import gi
```
Comment (theCapypara) [inline src/Library/demos/Avatar/main.py:1]: For some reasons the Python demo does not load the default avatar image for me, showing just a placeholder. But I can see in the inspector that the correct icon is placed there, so either something is wrong in my dev setup or this is an issue with the previewer process or PyGObject.

**Review by theCapypara — APPROVED** [review]

**Pull request merged**